script/fluentd.conf

```diff
@@ -25,6 +25,7 @@
 <match pubsub.publish>
     @type lm
     company_name "${LM_COMPANY_NAME}"
+    company_domain "${LM_COMPANY_DOMAIN}"
```

Comment:
Where is this used?

Reply:
The fluentd repository which is used by gcp to ingest logs.